claim/schema.py

```diff
@@ -111,52 +113,47 @@ def resolve_claim(self, info, id=None, uuid=None, **kwargs):
             return Claim.objects.get(uuid=uuid)
 
     def resolve_claims(self, info, **kwargs):
+        class AttachmentStatusEnum(Enum):
```

Comment:
Why is it defined here? If not defined already it should be added in claim model.

Reply:
no idea, I just put it up (it was in the middle of the funct)

Reply:
To be restructured later.